RELEASE-NOTES.md

```diff
@@ -0,0 +1,6 @@
+# Release notes
```

Comment:
Codacy found an issue: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=5870223)